### ISSUE TITLE
Blank disk label does not work

### ISSUE THREAD

**Issue by devinganger** [dalejos/Mikrotik-Script-Framework#1]
If the disk you have selected did not have a label created, then the script will not create the requested container.

**Comment by dalejos** [comment]
Correct, the script was designed to run with disks whose label is placed, this to be able to follow the disk after updating to ROS, since ROS would change the mount point for example from disk1 to disk2.
